server/src/main/java/io/druid/server/metrics/MetricsModule.java

```diff
@@ -95,4 +101,40 @@ public MonitorScheduler getMonitorScheduler(
         monitors
     );
   }
+
+  @Provides
+  @ManageLifecycle
+  public JvmMonitor getJvmMonitor(Properties props)
+  {
+    return new JvmMonitor(getDimensions(props));
+  }
+
+  @Provides
+  @ManageLifecycle
+  public JvmCpuMonitor getJvmCpuMonitor(Properties props)
+  {
+    return new JvmCpuMonitor(getDimensions(props));
+  }
+
+  @Provides
+  @ManageLifecycle
+  public SysMonitor getSysMonitor(Properties props)
+  {
+    return new SysMonitor(getDimensions(props));
+  }
```

Comment:
is dataSource and taskId relevant for SysMonitor? Wouldn't it report same thing independent of the jvm.